sixMans/queue.py

```diff
@@ -1,13 +1,14 @@
-import discord
 import collections
 import datetime
 import uuid
 from queue import Queue
+from typing import List
+
+import discord
 
-team_size = 6
 
 class SixMansQueue:
-    def __init__(self, name, guild, channels, points, players, gamesPlayed):
+    def __init__(self, name, guild: discord.Guild, channels: List[discord.TextChannel], points, players, gamesPlayed, teamSize):
```

Comment:
I think it would be safe to default team_size to six => `teamSize=6`

Reply:
I have that coming from the SixMans class where it's defaulted to 6. I think I'd prefer it be forced to come from that cog so we know every queue is using that original teamSize variable, if that makes sense.